src/compiler/get_candid_and_canister_methods.ts

```diff
@@ -7,7 +7,7 @@ import { generateCandidAndCanisterMethods } from './generate_candid_and_canister
 import { CandidGen, CanisterMethods, CompilerInfo } from './utils/types';
 
 export function getCandidAndCanisterMethods(
-    candidGen: CandidGen = 'automatic',
+    candidGen: CandidGen,
```

Comment:
Could we add the default http to here instead of where it was added

Reply:
Done